terraform/application/job.tf

```diff
@@ -0,0 +1,17 @@
+module "job_configuration" {
```

Comment:
lets call this module "migrations_job"
the application.tf changes will then be depends_on = [module.migrations_job]